nixos/modules/services/misc/portunus.nix

```
@@ -230,7 +230,10 @@ in
         description = "Self-contained authentication service";
         wantedBy = [ "multi-user.target" ];
         after = [ "network.target" ];
-        serviceConfig.ExecStart = "${cfg.package.out}/bin/portunus-orchestrator";
+        serviceConfig = {
+          ExecStart = "${cfg.package.out}/bin/portunus-orchestrator";
```

Could you make a new commit for this change?
```suggestion
          ExecStart = "${cfg.package}/bin/portunus-orchestrator";
```